pyracf/access/access_admin.py

```diff
@@ -38,15 +38,15 @@ def __init__(
         self,
         debug: bool = False,
         generate_requests_only: bool = False,
-        add_field_data: Union[dict, None] = None,
-        overwrite_field_data: Union[dict, None] = None,
+        update_existing_segment_traits: Union[dict, None] = None,
+        overwrite_segment_traits: Union[dict, None] = None,
```

Comment:
Change this to `overwrite_existing_segment_traits`

Reply:
this does feel more consistent. change made